commons-db/src/main/java/com/palantir/nexus/db/sql/SQL.java

```diff
@@ -50,7 +50,7 @@
  * All the methods deprecated in this class have alternatives in {@link PalantirSqlConnection}
  */
 public abstract class SQL extends BasicSQL {
-    static final Logger sqlExceptionlog = LoggerFactory.getLogger("sqlException." + SQL.class.getName()); //$NON-NLS-1$
+    private static final Logger sqlExceptionlog = LoggerFactory.getLogger("sqlException." + SQL.class.getName()); //$NON-NLS-1$
```

Comment:
Again, this is probably ok but we should see if we can easily check internal projects aren't using this.